app/models/shop/stock.rb

```diff
@@ -0,0 +1,14 @@
+class Shop::Stock < ApplicationRecord
```

Comment:
Why do we need a module for that if it's a normal join associations. Am I missing something? `ShopsStocks` wouldn't work here?

Reply:
It would work. 
But it's just a hierarchy. 
So when you see Shop::Stock you know that stock is useless without a shop. Am I wrong?